pkg/model/piped.proto

```diff
@@ -71,8 +71,12 @@ message Piped {
     ConnectionStatus status = 11 [(validate.rules).enum.defined_only = true];
     // Currently applied config.
     string config = 12;
+    // NOTE: Fields `cloud_provider` and `platform_provider` are deprecated.
+    // Using `deploy_targets` to specify where to deploy instead.
     // List of configured platform providers.
     repeated PlatformProvider platform_providers = 16 [deprecated=true];
+    // NOTE: Fields `cloud_provider` and `platform_provider` are deprecated.
+    // Using `deploy_targets` to specify where to deploy instead.
```

Comment:
Should this be above line L9? 👀

Reply:
Delete these line and it doesn't need to add to L9. Deprecation announce is explained on L74.